src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            required
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="checkbox"
+          value="off"
+          >
```

Comment:
```suggestion
          <input
            type="checkbox"
            name="checkbox"
            value="off"
          >
```
Поправь отступы **везде**